pb/api/v1/data.proto

```diff
@@ -164,3 +158,38 @@ message Resource {
   string name = 1;
   string link = 2;
 }
+
+message ListTabSummariesRequest {
+  // Scope defines the GCS bucket to read the results from.
+  string scope = 1;
+  // Name of the dashboard to fetch tab summaries for.
+  string dashboard = 2;
+}
+
+message ListTabSummariesResponse {
+	// List of tab summaries.
+	repeated TabSummary tab_summaries = 1;
+}
+
+message TabSummary {
+	// The name of the dashboard this tab belongs to.
+	string dashboard_name = 1;
+
+	// The name of the tab.
+  string tab_name = 2;
+
+  // Overall status for the dashboard tab (e.g. PASSING, FAILING, etc.)
+  string overall_status = 3;
+
+  // Summary of the status for this dashboard tab.
+  string detailed_status_message = 4;
+
+  // Timestamp at which tests last ran.
+  google.protobuf.Timestamp last_run_timestamp = 5;
+
+  // Timestamp at which the test group was last updated.
+  google.protobuf.Timestamp last_update_timestamp = 6;
+
+  // The ID for the latest passing build.
+  string latest_green = 7;
```

Comment:
nit: Maybe we want to do latest_passing_build instead for clarity?

Reply:
Done!